src/Uno.UWP/Devices/Geolocation/Geofencing/Geofence.cs

```diff
@@ -0,0 +1,38 @@
+#pragma warning disable 108 // new keyword hiding
+#pragma warning disable 114 // new keyword hiding
+
+using System;
+
+namespace Windows.Devices.Geolocation.Geofencing
+{
+	public partial class Geofence
+	{
+		public TimeSpan Duration { get; internal set; }
```

Comment:
You can remove the `internal set;` all fields are assigned through the constructor.